securetea/args/arguments.py

```diff
@@ -374,5 +375,20 @@ def get_args():
         help="URL on which operations are to be performed"
     )
 
+    parser.add_argument(
+        '--ids',
+        action="store_true",
+        required=False,
+        help="Start Intrusion Detection System (IDS)"
+    )
+
+    parser.add_argument(
+        '--threshold',
+        '-th',
+        type=int,
+        required=False,
+        help="Intrusion Detection System (IDS) threshold"
```

Comment:
Could you please provide the value range?

Reply:
Sure :)

Reply:
@rejahrehim Done! in my new PR. Please have a look. :)